src/observer/migrations/0013_auto_20180209_0611.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11 on 2018-02-09 06:11
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('observer', '0012_auto_20180122_0528'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='PressPackage',
+            fields=[
+                ('id', models.CharField(max_length=8, primary_key=True, serialize=False)),
+                ('title', models.CharField(max_length=255)),
+                ('published', models.DateTimeField()),
+                ('updated', models.DateTimeField()),
+                ('subjects', models.ManyToManyField(blank=True, help_text='subjects this press package mentions directly', to='observer.Subject')),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Profile',
+            fields=[
+                ('id', models.CharField(max_length=8, primary_key=True, serialize=False)),
+                ('name', models.CharField(max_length=255)),
```

Comment:
this migration doesn't match the `Profile` class defined earlier, which has minimal fields

Reply:
Or, well, there is a later migration that removes the columns... Is there a way to squash those?

Reply:
migrations are auto generated. 

it also seems to be a bit of state that messes with `--ci`. I had to manually go in and tweak the db as it had performed a migration that was later replaced and it complained it couldn't be done. least painful approach was just another migration.

yes, migrations can be squashed. I've squashed the lax migrations once and it was a little painful because of custom fields (`annoying.fields.jsonfield`). the performance only starts to become painful at about 30+ migrations.